src/naemon/naemon.c

```diff
@@ -529,6 +529,15 @@ int main(int argc, char **argv)
 		nerd_init();
 		timing_point("NERD initialized\n");
 
+		/* let the parent know we're good to go and that it can let go */
```

Comment:
Shouldn't we signal the parent after the child has checked the object configuration? I.e. after `result = read_all_object_data(config_file);`